docs/setup/install/troubleshooting/index.md

```diff
@@ -26,7 +30,7 @@ In addition, if your GKE cluster is a [private cluster](https://cloud.google.com
 
 ## Configuring Network Volume Accessor
 
-For network volume such as NFS, Stash needs to deploy a helper deployment in the same namespace as the Repository that uses the NFS as backend to provide Snapshot listing facility. We call this helper deployment network volume accessor. You can configure its resources, user id, privileged permission etc. Run the following command to enable network volume accessor,
+For network volume such as NFS, KubeStash needs to deploy a helper deployment in the same namespace as the Repository that uses the NFS as backend to provide Snapshot listing facility. We call this helper deployment network volume accessor. You can configure its resources, user id, privileged permission etc. Run the following command to enable network volume accessor,
```

Comment:
For network volumes like NFS, KubeStash requires deploying a helper deployment within the same namespace as the BackupStorage. This deployment mounts the NFS volume to access the necessary resources. We call this helper deployment `network volume accessor`. You can configure its resources, user id, privileged permission etc.  To enable the network volume accessor, run the following command: